dev/com.ibm.ws.jaxrs.2.0.cdi.1.2_fat/fat/src/com/ibm/ws/jaxrs20/cdi12/fat/test/DependentIntoJaxTest.java

```diff
@@ -24,7 +24,7 @@
 import componenttest.topology.impl.LibertyServer;
 
 @RunWith(FATRunner.class)
-@SkipForRepeat("EE9_FEATURES") //The fact this doesn't work on EE9 is probably a bug and needs investigating
+@SkipForRepeat({"EE9_FEATURES", "EE10_FEATURES"}) //The fact this doesn't work on EE9 is probably a bug and needs investigating
```

Comment:
FYI, I opened https://github.com/OpenLiberty/open-liberty/issues/23743 to track this